### PR TITLE
fix(csv): Handle empty data array

### LINKED ISSUE #233: artoo.saveCSV saves corrupted data when called on empty array
```js
artoo.saveCsv([], 'test.csv');
```

### PR DESCRIPTION
**Pull request by joscha**
This returns an empty string (empty CSV) if the given data array is empty.
Fixes #233.

### CODE REVIEW

**Comment by Yomguithereal** [comment]
Thanks @joscha. This seems like a sensible fix indeed. Can you just add a space between your `if` and the condition's first parenthesis so I can merge your PR please :smile:?

File: src/artoo.writers.js [hunk 1]
```diff
@@ -58,6 +58,9 @@
 
   // Converting an array of arrays into a CSV string
   function toCSVString(data, params) {
+    if(data.length === 0) {
```
Comment (Yomguithereal) [inline src/artoo.writers.js]: Right here.

**Comment by joscha** [comment]
done @Yomguithereal

**Comment by Yomguithereal** [comment]
Thanks @joscha. Merging right now.

**Comment by Yomguithereal** [comment]
Do you need a release soon?

**Comment by joscha** [comment]
Nope, to be honest I don't use artoo - I just saw the issue #233 and knew I could fix it easily :)

**Comment by Yomguithereal** [comment]
Thanks anyway :)

**Pull request merged**